src/Compilers/CSharp/Portable/Lowering/SyntheticBoundNodeFactory.cs

```diff
@@ -1373,7 +1392,7 @@ internal static BoundExpression NullOrDefault(TypeSymbol typeSymbol, SyntaxNode
             }
 
             var syntax = argument.Syntax;
-            var type = argument.Type;
+            var type = argument.Type!;
```

Comment:
>argument.Type!; [](start = 23, length = 15)

how do we know?

Reply:
Will add an assertion.

---
In reply to: [386590778](https://github.com/dotnet/roslyn/pull/42068#discussion_r386590778) [](ancestors = 386590778)